src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -53,9 +68,40 @@ export default {
 			default: ''
 		}
 	},
+	apollo: {
+		query: componentQuery,
+		preFetch(config, client) {
+			return client.query({
+				query: componentQuery
+			}).then(() => {
+				return client.query({
+					query: experimentAssignmentQuery, variables: { id: 'braintree_dropin_apple_google' }
+				});
+			});
+		},
+		result() {
+			// applePay & googlePay payment methods experiment - EXP-GROW-168-Aug2020
+			const paymentMethodsExperiment = this.apollo.readFragment({
+				id: 'Experiment:braintree_dropin_apple_google',
+				fragment: experimentVersionFragment,
+			}) || {};
+			if (paymentMethodsExperiment.version === 'shown') {
+				this.paymentTypes = ['paypal', 'card', 'googlePay'];
+			}
+			// Fire Event for EXP-GROW-168-Aug2020
+			if (paymentMethodsExperiment.version && paymentMethodsExperiment.version !== 'unassigned') {
```

Comment:
Once deployed, please verify this is firing correctly. We normally want to reserve analytics events until mounted or after...

Reply:
@mcstover good catch, fixed this